tests/test_emitter.py

```diff
@@ -428,6 +427,9 @@ def test_recursive_off():
     with pytest.raises(Empty):
         event_queue.get(timeout=5)
 
+    mkfile(p('b'))
```

Comment:
Could you also test folder and file creation in sub-sub-folder?

Reply:
Good point, was actually pondering about some additional scenarios that should be tested, most notably around file / directory moved events though.